### PR TITLE
Add main structure

### PR DESCRIPTION
**Pull request by FedericoBernal**
Fixes #

## Specific Changes

  - 
  -
  -

### CODE REVIEW

File: samples/25.message-reaction/pom.xml [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+      </dependency>
+      <dependency>
+        <groupId>org.junit.vintage</groupId>
+        <artifactId>junit-vintage-engine</artifactId>
+        <scope>test</scope>
+      </dependency>
+
+      <dependency>
+        <groupId>com.microsoft.bot</groupId>
+        <artifactId>bot-ai-qna</artifactId>
```
Comment (matiasroldan6) [inline samples/25.message-reaction/pom.xml:68]: does it require qna?

File: samples/25.message-reaction/pom.xml [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+        <version>4.6.0-preview9</version>
+      </dependency>
```
Comment (Batta32) [inline samples/25.message-reaction/pom.xml:70]: Do we really need qna for this sample?

**Review by Batta32 — CHANGES_REQUESTED** [review]

**Review by fran893 — CHANGES_REQUESTED** [review]
I think you missed to add the `DeploymentTemplates` and the `TeamsAppManifest` folders

File: samples/25.message-reaction/pom.xml (hunk 2 above)
Comment (FedericoBernal) [reply samples/25.message-reaction/pom.xml:70]: You are right, we don't need it in this sample. I will delete this

**Review by Batta32 — APPROVED** [review]
LGTM

**Review by fran893 — APPROVED** [review]

**Pull request merged**